spec/data/real_life_sample.yml

```diff
@@ -0,0 +1,14197 @@
+---
```

Comment:
I suggest not open-sourcing Sage internal i18n files but reduce the example file to an absolute minimum to clearly show what `sanitize_yaml` does.